content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md

```diff
@@ -2,3 +2,5 @@
 title: Create a Cluster
 weight: 10
 ---
+
+{{< glossary_definition term_id="Cluster" length="short" >}}
```

Comment:
Hi @RA489 .
Is the `term_id` attribute "cluster"?

Reply:
`term_id` attribute "cluster" doesn't render its definition.

Reply:
Hi @RA489 .
I ran a test of the glossary_definition shortcode in PR #29712.
Can you modify the glossary definition cluster attribute and test in your PR?

Reply:
sure.

Reply:
modified the glossary definition cluster attribute